src/main/scala/gitbucket/core/api/ApiMilestone.scala

```diff
@@ -0,0 +1,52 @@
+package gitbucket.core.api
+
+import gitbucket.core.model.{Milestone, Repository}
+import gitbucket.core.util.RepositoryName
+import java.util.Date
+
+/**
+ * https://docs.github.com/en/rest/reference/issues#milestones
+ */
+case class ApiMilestone(
+  url: ApiPath,
+//  html_url: ApiPath,
```

Comment:
Why can't we provide `html_url`?

Reply:
@takezoe I thought gitbucket does not have url like `/:owner/:repository/milestone/:id`. So I didn't provide...
Maybe we can provide `/owner/:repository/issues?milestone={name}&state=open`  ...?
.

Reply:
For now, `/owner/:repository/issues?milestone={name}&state=open` is the only possible URL for milestones. However, we may need to have anothero URL which shows the status of the milestone.

Reply:
@takezoe fixed PR.  Now use `/owner/:repository/issues?milestone={name}&state=open` for `html_url`.
> we may need to have anothero URL which shows the status of the milestone.
Yeah it would be nice if GitBucket has each milestone page like GitHub that would be another issue.